### PR TITLE
entity_info: allow to see the exact origin of the brush entities

### PR DESCRIPTION
**Pull request by SmileyAG**
Old behaviour:

изображение

New behaviour:

изображение

### CODE REVIEW

File: BunnymodXT/hud_custom.cpp [hunk 1]
```diff
@@ -773,13 +773,16 @@ namespace CustomHud
 					}
 				}
 
+				// Borrowed from https://github.com/SNMetamorph/goldsrc-monitor/blob/08c368e246d09996b2d85e4367d4d8cc1e507712/sources/library/displaymode_entityreport.cpp#L45-L48
+				const Vector origin = ent->v.origin + ((ent->v.mins + ent->v.maxs) / 2.f);
```
Comment (YaLTeR) [inline BunnymodXT/hud_custom.cpp:777]: I think this should only be done for brush entities. For entities with an actual origin it should show the origin as is.

File: BunnymodXT/hud_custom.cpp [hunk 2]
```diff
@@ -777,9 +777,17 @@ namespace CustomHud
 				{
 					out << "Yaw: " << ent->v.angles[1] << '\n';
 
-					out << "X: " << ent->v.origin.x << '\n';
-					out << "Y: " << ent->v.origin.y << '\n';
-					out << "Z: " << ent->v.origin.z << '\n';
+					// Borrowed from https://github.com/SNMetamorph/goldsrc-monitor/blob/08c368e246d09996b2d85e4367d4d8cc1e507712/sources/library/displaymode_entityreport.cpp#L45
+					Vector origin;
+
+					if (strstr(classname, "func_") != NULL)
```
Comment (YaLTeR) [inline BunnymodXT/hud_custom.cpp:783]: this is a pretty cursed way to check for this

Comment (SmileyAG) [reply BunnymodXT/hud_custom.cpp:783]: How I should do check that, then? According to TWHL, most of brush entities is `func_` or `trigger_`: https://twhl.info/wiki/page/category%3ABrush_Entities

Comment (SmileyAG) [reply BunnymodXT/hud_custom.cpp:783]: @YaLTeR Oh, I think if I'll be check for `ent->v.solid == SOLID_BSP`, would be that works?

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**